src/plugins/options/options.ts

```diff
@@ -75,6 +56,32 @@ export class UIPOptions extends UIPPlugin {
     this.appendChild($theme);
   }
 
+  protected renderSettingsControl() {
+    const $settings = document.createElement('div');
+    CSSClassUtils.add($settings, 'uip-option');
+    const settingsControlId = randUID();
+    $settings.innerHTML = `
+        <div class="option-item">
+            <input type="checkbox" id=${settingsControlId}-settings-control name=${settingsControlId}-control
+            class="option-checkbox" settings-control='settings'>
+            <label class="option-label" for=${settingsControlId}-settings-control>Settings</label>
+        </div>`;
+    this.appendChild($settings);
+  }
+
+  protected renderEditorControl() {
+    const $editor = document.createElement('div');
+    CSSClassUtils.add($editor, 'uip-option');
+    const editorControlId = randUID();
+    $editor.innerHTML = `
+        <div class="option-item">
+            <input type="checkbox" id=${editorControlId}-settings-control name=${editorControlId}-control
+            class="option-checkbox" editor-control='editor'>
+            <label class="option-label" for=${editorControlId}-settings-control>Editor</label>
+        </div>`;
+    this.appendChild($editor);
+  }
+
```

Comment:
I think that it definitely should be simplified to one method with just passing the appropriate option name. but am i right, it 's planning to be refactored eith new otions architecture?